__mocks__/svgrMock.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default 'SvgrURL';
+export const ReactComponent = 'div';
```

Comment:
This is [recommended per docs](https://react-svgr.com/docs/jest/).

Reply:
Yep, correct 👍 

But can we move those mocks to `tests/js/__mocks__` instead so they live next to the config? Highly appreciated.

Reply:
Ah right, Jest mocks. https://jestjs.io/docs/en/manual-mocks

Well then 👍